_docs/technical/2017-01-04-updater.md

```diff
@@ -111,23 +120,27 @@ Please refer to the [core types
 module](https://github.com/input-output-hk/cardano-sl/blob/22360aa45e5dd82d0c87872d8530217fc3d08f4a/src/Pos/Update/Core/Types.hs)
 for more information.
 
-## Update Proposal Adoption
+## Update Proposal Approval
 
 A very important part of implementation of the update mechanism is
 the part that works with genesis blocks for epochs and applies updates.
```

Comment:
Applying an update happens after it is confirmed and downloaded. Download procedure is described later. I think update application is not in scope of this section.